Explorer/Assets/DCL/UserInAppInitializationFlow/LoadingStatus.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using DCL.Utilities;
+using System.Collections.Generic;
+using DCL.DebugUtilities.UIBindings;
+using Utility;
+
+namespace DCL.UserInAppInitializationFlow
+{
+    public class LoadingStatus : ILoadingStatus
+    {
+        public ReactiveProperty<LoadingStage> CurrentStage { get; } = new (LoadingStage.Init);
+        public ElementBinding<string> CurrentStageBinding { get; }= new (string.Empty);
+        public ElementBinding<string> CurrentAssetsStateBinding { get; } = new (string.Empty);
+
+        private Action<LoadingStage> analyticsCallback;
```

Comment:
following to our design, instead of having `AnalyticsCallback` we need to create a decorated loading status (`LoadingStatusAnalyticsDecorator`) that will subscribe probably in constructor and unsubscribe whenever `== LoadingStage.Completed`. So basically, logic for lines 73-74 and 81-85 should be extracted there.  Let's still use this decorated class even after analytics unsubscribing, just to keep things a bit easier.